### DISCUSSION TITLE
Feature: Show progress for magit-clone

### DISCUSSION THREAD

**Discussion by federicotdn** [magit/magit#5251] [Ideas]
When using Magit to e.g. pull from a repo, I can press `$` from the status buffer to see the output of the `git` command, which can give an idea of how the process is going. For `magit-clone` however, I have not found an equivalent for this, which would be helpful specially when cloning very large repos.

I can see the `git` process listed by using `M-x list-processes` though. Maybe Magit could bring up a buffer showing the output of `git clone` when running `magit-clone`? I tried also looking to see if such a buffer exists already but was maybe buried, but I couldn't find one.

**Comment by tarsius** [comment]
Magit calls `git` in a way that prevents that from showing progress indicators. Changing that would be hard and likely not worth the effort.

While `git clone` is running `git clone` is shown in the mode-line. This will have to do as indicator that that command is in fact still running (as opposed of, e.g., having already failed for some reason). Same for other `git` commands "run for side-effects".

**Comment by federicotdn** [comment]
That makes sense. I did notice now the `magit-process` buffer where `git clone` is running (hadn't found it before for some reason), that is already useful even if no actual percentage is being displayed 👍🏻
